snap-build/config_s390x.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 IBM
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+local arch_qemu="s390x"
+local arch_image="bionic-server-cloudimg-s390x.img"
+local arch_image_url="https://cloud-images.ubuntu.com/bionic/current/${arch_image}"
```

Comment:
I've only just spotted this duplication. Not a problem for this PR as you're copying the same approach as the current architectures but I've raised #349 so we can avoid this.